src/main/webapp/resources/js/apis/linker/linker.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+import { setBaseUrl } from "../../utilities/url-utilities";
+
```

Comment:
Instead of setting the base url in the POST call we should create a const before the function which will setbaseurl to `ajax/ngs-linker`. Just in case we need to add more to the file in the future.

Reply:
Sure sounds good, updated in 52966a528b82fba549c91cc4d897647daa896a5f